### PR TITLE
Adds home page and fixes style of campaign button

### PR DESCRIPTION
**Pull request by erik0686**
### What does this PR do?

* fixes style of campaign index view to match others
* adds home view

### CODE REVIEW

File: app/assets/stylesheets/campaigns.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	width: 20%;
+}
+
+.skill-list {
+	list-style-type: none;
+	position: relative;
+	right: 40px;
+}
+
+.space_between_image {
```
Comment (kurenn) [inline app/assets/stylesheets/campaigns.scss]: You were using dash case and suddenly decided to go with snake case?

File: app/assets/stylesheets/home.scss [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	display: inline-block;
+	font-family: 'Poller One', cursive;
+    margin-left: 80px;
+    margin-right: 80px;
+	padding-top: 30px;
+	
+}
+
+a:link {
+    color: black;
```
Comment (kurenn) [inline app/assets/stylesheets/home.scss]: Never use named colors, instead, use hex values

File: app/assets/stylesheets/home.scss [hunk 3]
```diff
@@ -0,0 +1,30 @@
+img.main-img {
+	display: block;
+    margin-left: auto;
+    margin-right: auto;
+}
+.links {
+    color: #C09448;
```
Comment (kurenn) [inline app/assets/stylesheets/home.scss]: Check for indentation

File: app/controllers/home_controller.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+class HomeController < ApplicationController
+	def index
+
```
Comment (kurenn) [inline app/controllers/home_controller.rb]: Remove this space

File: app/helpers/home_helper.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+module HomeHelper
```
Comment (kurenn) [inline app/helpers/home_helper.rb]: Remove this empty file

**Review by kurenn — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/home.scss [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    display: inline-block;
+    font-family: 'Poller One', cursive;
+    margin-left: 80px;
+    margin-right: 80px;
+    padding-top: 30px;
+}
+
+a:link {
```
Comment (kurenn) [inline app/assets/stylesheets/home.scss]: From line 15 to 27, you can use the sass nesting feature, something like:

```sass
a {
 color:  #000;

 &:hover {
   color: #000;
 }
....
}
```

**Review by kurenn — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/home.scss (hunk 6 above)
Comment (erik0686) [reply app/assets/stylesheets/home.scss]: Ok, I will change it

**Review by mayra-cabrera — CHANGES_REQUESTED** [review]
@erik0686 Be consistent with your CSS files indentation settings, some of them have four spaces and other ones have two spaces

File: app/assets/stylesheets/home.scss [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+  padding-top: 30px;
+}
+
+a {
+  color: #000000;
+
+  &:active {
+    color: #000000;
+  }
+  &:hover {
```
Comment (kurenn) [inline app/assets/stylesheets/home.scss]: Add a line between the pseudoselectors

**Review by kurenn — APPROVED** [review]

File: app/assets/stylesheets/home.scss [hunk 8]
```diff
@@ -0,0 +1,36 @@
+img.main-img {
+  display: block;
+  height: 50%;
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
+}
```
Comment (kurenn) [inline app/assets/stylesheets/home.scss:7]: Add a separation in here please

**Review by kurenn — CHANGES_REQUESTED** [review]

**Pull request merged**